paper/user_study.tex

```diff
@@ -49,8 +49,128 @@ \subsection{Results}
+        \cline{2-3}
+        & 11. After performing an undo or redo action, the cursor should jump to the appropriate location to reflect the changes made by the user.& 10. Will be consider for future work. \\
+        \hline
+    \end{tabular}
+    \caption* {User Study Feedback and Resolution (Continued)}
+\end{table*}
+
+
+\clearpage % Force a page break
+
+\begin{table*}
+
+    \centering
+    \small % Reduce font size
+    \setlength{\extrarowheight}{5pt} % Add extra vertical space
+    \renewcommand{\arraystretch}{1.5} % Adjust the vertical spacing between rows
+    \begin{tabular}{|c|p{0.4\linewidth}|p{0.4\linewidth}|}
+        \hline
+        User Study 2 & 1. A graph-based approach would be more intuitive for handling complex settings. & 1. Will be considered for future work. For now, we have a tree-based approach rather than a graph-based one. \\
+        \cline{2-3}
+        & 2. Providing immediate feedback to users when they enter incorrect ranges is essential to prevent them from inputting invalid values into the property. & 2. Will be consider for future work. For now, a red underline serves as an indicator to direct the user's attention when something is wrong. Additionally, when hovering over it, a dialog box can be used to inform the user of specific conditions. \\
+        & 3. Validation errors should also be reflected in the GUI panel, including for child properties. & 3. Currently, invalid properties are indicated in the code panel using a red cross mark, while in the GUI editor panel, invalid properties are highlighted with a red underline. \\
+        \cline{2-3}
+        & 4. When dealing with an array, the naming format for object name should be improved, replacing the default numerical labels (0, 1, 2, etc.) with a more descriptive way of showing the object. & 4. Improvement can be achieved by replacing simple numerical labels (0, 1, 2, etc.) for objects with a naming convention like "PropertyName[0]" and so forth. \\
+        \cline{2-3}
+        & 5. Additionally, when clicking a new property, it should not automatically create a new field, especially when there is already one field by default for adding the first property. & 5. Will be Consider for future work. \\
+        \cline{2-3}
+        & 6. In the schema editor, it would be more consistent and user-friendly if the functionality to give a name to a property were located on the right side of the GUI editor panel, eliminating the need to click on the property first in order to rename it. & 6. We have retained the property naming on the right end but improved it with an intuitive design. When clicking on a new property in the schema editor, it is now highlighted to prompt the user to change the name. \\
```

Comment:
```suggestion
        & 6. In the schema editor, it would be more consistent and user-friendly if the functionality to give a name to a property were located on the right side of the GUI editor panel, eliminating the need to click on the property first in order to rename it. & 6. Because JSON schema does not allow to specify the name of a property is a field value, we have retained the property naming on the right side but improved it with a more intuitive design. When clicking on a new property in the schema editor, it is now highlighted to prompt the user to change the name and a pencil icon next to the property indicates that it can be renamed when clicking that icon.   \\
```